descriptors/ModuleDescriptor-template.json

```diff
@@ -110,6 +110,14 @@
             "orders-storage.vendor_details.item.delete"
           ]
         },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/orders/order-lines",
+          "permissionsRequired": ["orders.po-lines.collection.get"],
+          "modulePermissions": [
+            "orders-storage.po_lines.collection.get"
```

Comment:
`po-lines` not `po_lines`

Reply:
I've copied that from mod-orders-storage permissions set  as is. If we change naming here, so permission naming in mod-orders-storage also will be needed to update. Do we need these updates?

Reply:
I see.  No, not as part of this Story/PR.  Go ahead with this as is.